ci/docker/install/ubuntu_onnx.sh

```diff
@@ -27,8 +27,17 @@ set -e
 set -x
 
 echo "Installing libprotobuf-dev and protobuf-compiler ..."
-apt-get install -y libprotobuf-dev protobuf-compiler
+apt-get install -y autoconf automake libtool curl make g++ unzip
+git clone --recursive https://github.com/google/protobuf.git
+cd protobuf
+git checkout 80a37e0782d2d702d52234b62dd4b9ec74fd2c95
+./autogen.sh
+./configure --prefix=/usr
+make
+make check
+make install 
+ldconfig
 
 echo "Installing pytest, pytest-cov, protobuf, Pillow, ONNX and tabulate ..."
-pip2 install pytest==3.4.0 pytest-cov==2.5.1 protobuf==3.0.0 onnx==1.0.1 Pillow==5.0.0 tabulate==0.7.5
-pip3 install pytest==3.4.0 pytest-cov==2.5.1 protobuf==3.0.0 onnx==1.0.1 Pillow==5.0.0 tabulate==0.7.5
+pip2 install pytest==3.4.0 pytest-cov==2.5.1 protobuf==3.4.0 onnx==1.1.0 Pillow==5.0.0 tabulate==0.7.5
+pip3 install pytest==3.4.0 pytest-cov==2.5.1 protobuf==3.4.0 onnx==1.1.0 Pillow==5.0.0 tabulate==0.7.5
```

Comment:
Why still install protobuf if we're compiling it ourselves

Reply:
This is just installing the python bindings, while compiling it we are building and installing these two libraries - libprotobuf-dev and protobuf-compiler.

Reply:
I see, thanks